packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -90,12 +90,13 @@ export async function createConsolidatedFromConversions({
     log(errorToString(e));
   }
 
+  // TODO(2025-05-20|ENG-328): Before continuing to make changes to this command and duplicative operations,
```

Comment:
super nit: no need for the date as it shows up on blame? and it doesn't add much IMO :)

Reply:
Going to leave it - i tend to add the date because when people refactor and move stuff around, the git blame will change, and can make it look like the TODO is more recent than it is! This helps us be more explicit about when this was noted, I've found them really helpful for deciding whether to look into the comment further (has it been a year or a month?)

Reply:
Fine, I just don't see what value can we get from the "one year" vs "one month" - so for me it's just noise.